internal/pkg/service/common/etcdop/op/txn.go

```diff
@@ -0,0 +1,165 @@
+package op
+
+import (
+	"context"
+
+	"go.etcd.io/etcd/api/v3/etcdserverpb"
+	etcd "go.etcd.io/etcd/client/v3"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+// TxnOp provides high-level interface above etcd.Txn.
+// Values are mapped by the op.mapper operations
+// and processed by the op.processors, see ForType[R] type.
+type TxnOp struct {
+	cmps    []etcd.Cmp
+	thenOps []Op
+	elseOps []Op
+}
```

Comment:
V tomto commite som pridal high-level podporu transakcii.

Asi nie je velmi co komentovat, funguje to rovnako ako etcd transakcie.
Rozdiel je v tom, ze responses su namapovane, rovnako ako pri ostatnych operaciach definovanych v tomto pkg.